chromium_src/chrome/browser/themes/theme_service_win.h

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Include all headers in chrome/browser/themes/theme_service_win.h to
+// prevent applying newly defined DISALLOW_COPY_AND_ASSIGN macro.
+// If not, many static_assert are catched.
+#include "base/optional.h"
+#include "base/win/registry.h"
+#include "chrome/browser/themes/theme_service.h"
+
+#include "brave/browser/themes/brave_theme_service.h"
+#include "brave/common/brave_override_util.h"
+
+#undef DISALLOW_COPY_AND_ASSIGN
+#define DISALLOW_COPY_AND_ASSIGN(TypeName)                                                    \
+  DISALLOW_COPY(TypeName);                                                                    \
+  DISALLOW_ASSIGN(TypeName);                                                                  \
+  static_assert(strings_equal(#TypeName, "ThemeServiceWin"), "Use only for ThemeServiceWin"); \
```

Comment:
just leaving a comment here from our DM discussion to make sure we don't forget to change this to selectively apply `friend` instead of asserting

Reply:
Will be reverted to patching `friend class`. I can't find how conditionally add in macro.

Reply:
I concur. Not worth the time right now